src/analyses/callstringTracking.ml

```diff
@@ -0,0 +1,36 @@
+open Analyses
+
+module Spec : MCPSpec =
+struct
+  include Analyses.IdentitySpec
+
+  let name () = "callstringTracking"
+
+  module FundecSet = SetDomain.Make (CilType.Fundec)
+  module FundecList = Lattice.Flat(Printable.Liszt(CilType.Fundec))
```

Comment:
Would it be possible to reuse the modules from `callstring` here to make this configurable?